### DISCUSSION TITLE
Different anchor points for measurement data points corresponding to different output variables

### DISCUSSION THREAD

**Discussion by gnodking7** [lululxvi/deepxde#790] [Q&A] [answered]
Hello,

I'm working with a single time-dependent PDE problem with two outputs `Q(x,t)` and `S(x,t)` to learn.
I have some empirical/measurement data points for both `Q(x,t)` and `S(x,t)` that are to be included in the training process,
but I would like to do this such that I use **different spatio-temporal locations (different anchor points)** for `Q(x,t)` and `S(x,t)`.

I have no problem with training using `Q(x,t)`, `S(x,t)` measurement data points at the same spatio-temporal locations.
In this case, I use `X` to denote the locations shared by both `Q(x,t)`, `S(x,t)` measurements, and I use the following codes:

```
observe_Q = dde.icbc.PointSetBC(X, Qvals, component=1)
observe_S = dde.icbc.PointSetBC(X, Svals, component=0)

data = dde.data.TimePDE(geomtime, pde, [observe_S, observe_Q], num_domain=3540, num_boundary=160, num_initial=320, anchors=X)
```

But how to do this for two different spatio-temporal locations, say `X` and `Y`?
I've tried doing the following, but I get an error that my implementation with `anchors=[X, Y]` is incorrect.
```
observe_Q = dde.icbc.PointSetBC(X, Qvals, component=1)
observe_S = dde.icbc.PointSetBC(Y, Svals, component=0)

data = dde.data.TimePDE(geomtime, pde, [observe_S, observe_Q], num_domain=3540, num_boundary=160, num_initial=320, anchors=[Y, X])
```

**Comment by lululxvi** [comment, marked as answer]
You don't need `anchors` for `PointSetBC`. Everything else looks good.
